acceptance/lib/helper.rb

```diff
@@ -259,7 +285,34 @@ def https_request(url, request_method, cert, key, body = nil)
     response = http.request(request)
   end
 
-
+  def hup_server(host = master, timeout = 30)
+    pid=on(host, 'cat /var/run/puppetlabs/puppetserver/puppetserver').stdout.chomp
+    on(host, "kill -HUP #{pid}")
+    url="https://#{host}:8140/puppet/v3/status"
+    cert=get_cert(master)
+    key=get_key(master)
+    response_code = "0"
+    sleeptime = 1
+    while response_code != "404" && timeout > 0
+      sleep sleeptime
+      begin
+        response = https_request(url, 'GET', cert, key)
```

Comment:
Not sure it matters here, but I coded `https_request` to use keywords for the method parameter, thus:
`https_request(url, :get, cert, key)`